crates/sui-network/README.md

```diff
@@ -0,0 +1,11 @@
+# sui-network
+
+## Changing an RPC service
```

Comment:
Most will need to familiarize themselves with the towers service, and some will want to look into the tonic channel, route, etc. It would be nice to link to places where one might do each of those two things while this README is still barebones.

e.g.https://github.com/hyperium/tonic/blob/master/examples/routeguide-tutorial.md